doc/nrf/ug_radio_fem.rst

```diff
@@ -62,6 +62,59 @@ Before you add the devicetree node in your application, complete the following s
    See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
 #. Enable support for MPSL implementation in |NCS| by setting the :kconfig:option:`CONFIG_MPSL` Kconfig option to ``y``.
 
+Using FEM power models
+----------------------
+
+When a protocol driver requests a given transmission power to be output, MPSL splits the power into components - the SoC Power
+and the FEM gain.
+This gain is considered constant and accurate even though external conditions such as temperature might affect the effective gain achieved by the Front-End Module.
+
+To allow performing a split in a different way (for example to compensate for external conditions) a FEM power model can be used.
+You can use either one of the provided built-in models or provide your own custom model.
+
+In order to use this feature set the :kconfig:option:`CONFIG_MPSL_FEM_POWER_MODEL` Kconfig option to ``y``  and select one of the built-in
+models or provide a custom model.
+
+Using nRF21540 GPIO SPI built-in power model (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This feature is in experimental stage and there is no guarantee that it will improve or even not worsen the current operation.
+
+In order to use this model set :kconfig:option:`CONFIG_MPSL_FEM_POWER_MODEL` and :kconfig:option:`CONFIG_MPSL_FEM_POWER_MODEL_NRF21540_USE_BUILTIN` to ``y``.
+
+When this feature is used a model is used to compensate the FEM gain for the external conditions: temperature and FEM supply voltage as well as the
+carrier frequency and the FEM input power. It is assumed that the FEM supply voltage is constant. In order to inform the MPSL subsystem about
+this voltage, use the :kconfig:option:`CONFIG_MPSL_FEM_POWER_VOLTAGE` option.
+
+
+Adding custom power models
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the way MPSL splits the TX power into components does not meet the users requirements or the user wishes to implement his own compensation model
+a custom model implementation can be provided. In order to use such model the following steps have to be completed:
```

Comment:
I wouldn't speculate about the reasons why a user would like to use own power model. So what about
```suggestion
If a user wishes to implement own compensation model, the user can provide a custom model implementation.
To use such model complete following steps:
```

Reply:
'does not meet users requirements' is definitely something we could rephrase it by 'wishes to implement their own compensation model'.